### PR TITLE
New design proposal

### PR DESCRIPTION
**Pull request by danielsarsi**
Homepage

Some tidbits:
- Uses default GitHub ribbon;
- A nice moving background with screenshots;
- A dynamic button to download stable version for the current platform;
- Works on Chrome, Safari and Firefox (Edge should work too, but I haven't tested it).

Any thoughts?

### CODE REVIEW

**Comment by IntelOrca** [comment]
Very nice, I like it.

Few things:
- The background does not scroll in Edge.
- The background moves very slowly on high resolutions (chrome). Is it possible to hardware accelerate this? If not maybe a video would be better.
- You can see the background wrap on high resolutions.
- We probably want to point release builds to http://openrct2.org which has Windows, OSX, Linux for 0.0.3.1 stable and develop.

**Comment by danielsarsi** [comment]
I'm going to take a look on Edge very shortly.

The background works this way:
1. It contains 4 screenshots, 1080p;
2. In 30 seconds, it animates from left to right;
3. In the last 3 seconds, it fades way;
4. The screenshot covers the screen on resolutions bigger than 1080p, and only fades away;

Originally, I wanted to put an .mp4 of the intro, but FPS dropped a lot while recording it in 1080p, and I don't know how big this file can be. If someone can provide it, I can give it a try.

We can increase the resolution in the screenshots, this way it will move even in higher resolutions.

I didn't know OpenRCT2.org was ready, that's fantastic! I'm going to point to it.

**Comment by danielsarsi** [comment]
Slightly updated it:
Update
- Background works fine now in higher resolutions;
- Added a footer with some links;
- New development build button, pointing to OpenRCT2.org;
- Change platform by clicking.

I tested in Edge, it didn't worked, but after an F5, it did. I don't know what happened. Theoretically, it has to work.

Edge handles the animation perfectly, it blends pixels while it's animating, while WebKit and Blink handle it pixel by pixel. I can try a workaround, if you decide to keep the screenshots. Maybe I can put a limit for the moving animation to work, avoiding sluggishness in higher resolutions.

Let me know what you think.

File: index.html [hunk 1]
```diff
@@ -1,23 +1,123 @@
...
+    <ul id="buttons">
+      <li>
+        <a class="link" href="https://github.com/OpenRCT2/OpenRCT2/releases/download/v0.0.3.1/OpenRCT2-0.0.3.1-windows.zip">Download Stable</a>
+        <div id="information">
+          <span class="version">0.0.3.1</span> — <a class="platform" href="#" alt="Change platform…">Windows</a> — <span class="size">12.3 MB</span>
+        </div>
+      </li>
+      <li>
+        <a href="https://openrct2.org/downloads/latest/develop">Download Development Build</a>
+        <div id="information">Windows, Linux & OS X</div>
```
Comment (IntelOrca) [inline index.html]: Use `&amp;`

File: index.html [hunk 2]
```diff
@@ -1,23 +1,123 @@
...
+        link: 'https://github.com/OpenRCT2/OpenRCT2/releases/download/v0.0.3.1/OpenRCT2-0.0.3.1-linux.tar.xz',
+        size: 6288140, //bytes
+        version: '0.0.3.1'
+      },
+//      osx: {
+//        name: 'OS X',
+//        link: '',
+//        size: 0,
+//        version: ''
+//      }
```
Comment (IntelOrca) [inline index.html:36]: Why is OSX commented out?

Comment (danielsarsi) [inline index.html:36]: Because there's no stable build for it.

**Comment by IntelOrca** [comment]
- Edge works now and the performance is much better than Chrome.
- Chrome is still very slow at high resolutions.
- If the height of the browser window is low, the bottom bar overlaps the buttons. I think the logo needs resizing on lower resolution sizes.

**Comment by danielsarsi** [comment]
It really needs higher resolution screenshots for it too work nicely in Chrome. Yet, even bigger images would slow down the page load time. So, I put a limit for the moving background animation: if the screen width is bigger than 1420px, only fade out will run.

A video would be nice to have, it's more simple and pretty.

**Comment by danielsarsi** [comment]
Hold on, maybe a friend can record it.

**Comment by danielsarsi** [comment]
I've added the video background. It's 30 seconds long, 1920x1080, 9,6MB.
If you think the quality is too poor, let me know.

**Comment by IntelOrca** [comment]
Good, that is much better.

**Pull request merged**